docker-compose.yaml

```diff
@@ -58,6 +58,20 @@ services:
       interval: 30s
       timeout: 10s
       retries: 5
+  archive-mock:
+    build: https://github.com/SwissOpenEM/ScicatArchiveMock.git
+    depends_on:
+      rabbitmq:
+        condition: service_healthy
+      backend:
+        condition: service_started
+    environment:
```

Comment:
could you use a map for the envs? just for consistency with the other service where we use environment vars

Reply:
Of course!